js/libs/core.js

```diff
@@ -322,6 +322,15 @@ Fliplet.FormBuilder = (function() {
         'props', 'computed'
       ]), component);
 
+      component.methods.includesComponent = function(container, value) {
```

Comment:
Where is this function used?

Reply:
Fixed. This function was used by me earlier, at the moment I have removed it.